book-1-orientation/chapters/ADONET_INTRO.md

```diff
@@ -7,6 +7,7 @@ This will be an interactive lesson. Your instructor will lead you through follow
 1. Open the [departments and employees](./assets/departments-employees.sql) SQL script and copy it into Azure Data Studio. Select the entire contents of the file and run it. This will create the database, and the tables, and insert some data.
 1. In Visual Studio, create a new console application called `DepartmentsEmployees`.
 1. In your terminal, navigate to the directory where you created your project. The directory will have a `DepartmentsEmployees.sln` file in it.
+1. `cd` into your project directory. When you list what's in the directory, you should see your `Program.cs`.
```

Comment:
Please also mention that the `DepartmentsEmployees.csproj` file is in the directory along with the `Program.cs`.